.gitignore

```diff
@@ -1 +1,2 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
+.env
```

Comment:
I might suggest we change the `.env` [here](https://github.com/astronomer/ask-astro/blob/main/.gitignore#L7) to make it ignore everywhere (or has it already ignored it? 🤔 )

Reply:
haha, there wasn't even a `.gitignore` file when I made this PR